src/Microsoft.ML/MemoryCollection.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+
+namespace Microsoft.ML
+{
+    public class MemoryCollection
+    {
+        /// <summary>
+        /// Creates memory collection loader.
```

Comment:
>loader [](start = 38, length = 6)

loader from IList

Reply:
Is that not obvious from the parameter type? Perhaps adding a "from <paramref cref="data" />." or somesuch would be fine (added to both), but even that I'd say would be a bit too needlessly verbose.

---
In reply to: [187221474](https://github.com/dotnet/machinelearning/pull/106#discussion_r187221474) [](ancestors = 187221474)

Reply:
it's confusing that it says "Collection Loader" while it takes a List . perhaps we should rename it  to a ListLoader.  and split away from the StreamingEnumerableLoader  

---
In reply to: [187230849](https://github.com/dotnet/machinelearning/pull/106#discussion_r187230849) [](ancestors = 187230849,187221474)

Reply:
User no longer see class, it just get interface, is it better?

---
In reply to: [187238467](https://github.com/dotnet/machinelearning/pull/106#discussion_r187238467) [](ancestors = 187238467,187230849,187221474)